### PR TITLE
1 front end nextjs

### PR DESCRIPTION
**Pull request by jekiquin**
This is going to be a big one as it is the scaffolding for the repo

DoD: scaffolding

1. Nextjs
2. Tailwindcss
3. Typescript
4. Prettier
5. Eslint
6. Husky + lint-staged

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# [Virgl Capstone Spring 2023]
```
Comment (yhus) [inline README.md:1]: I'd recommend not mentioning anything related to Virgl or capstone

File: package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "virgl-capstone-fe",
```
Comment (yhus) [inline package.json:2]: Same here

**Review by yhus — APPROVED** [review]
Minor updates - otherwise LGTM!

File: README.md (hunk 1 above)
Comment (jekiquin) [reply README.md:1]: For sure. Will change to Turbocharger Frontend. It's still a placeholder, we'll probably change that once we have pretty clear idea on the name of the product and the description.

File: package.json (hunk 2 above)
Comment (jekiquin) [reply package.json:2]: Will change to turbocharger-frontend on here too

**Review by chunloy — APPROVED** [review]
Looks good to me!

**Pull request merged**